lib/std/crypto/siphash.zig

```diff
@@ -46,8 +46,6 @@ fn SipHashStateless(comptime T: type, comptime c_rounds: usize, comptime d_round
 
     return struct {
         const Self = @This();
-        const block_length = 64;
-        const digest_length = 64;
```

Comment:
These are observable from outside siphash.zig. They should be `pub`.

Reply:
https://discord.com/channels/697519923273924658/713755801583616100/929803897252511794

Reply:
Can you post the content here?